docs/theming.md

```diff
@@ -46,10 +46,7 @@ When you want more customization than a pre-built theme offers, you can create y
 A theme file is a simple Sass file that defines your palettes and passes them to mixins that output
 the corresponding styles. A typical theme file will look something like this:
 ```scss
-@import '~@angular2-material/core/theming/theming';
-@import '~@angular2-material/core/theming/palette';
-@import '~@angular2-material/core/core';
-@import '~@angular2-material/button/button-theme';
+@import '~@angular2-material/core/theming/all-theme';
```

Comment:
Isn't this (and the rest of the references in theming.md) now '~@angular/material...'?